### PR TITLE
Make CHAMP collections available to the Gradle runtime

### PR DESCRIPTION
**Pull request by bamboo**
See https://github.com/usethesource/capsule for more details.

### CODE REVIEW

**Comment by bamboo** [comment]
@bot-gradle test QFL

**Comment by bamboo** [comment]
@bot-gradle test RFN

File: build-logic/dependency-modules/src/main/kotlin/gradlebuild/modules/extension/ExternalModulesExtension.kt [hunk 1]
```diff
@@ -185,6 +186,7 @@ abstract class ExternalModulesExtension {
         bouncycastlePgp to License.MIT,
         bouncycastleProvider to License.MIT,
         bsh to License.Apache2,
+        capsule to License.BSDStyle,
```
Comment (ljacomet) [inline build-logic/dependency-modules/src/main/kotlin/gradlebuild/modules/extension/ExternalModulesExtension.kt:189]: Is that enough to have it visible in the docs and / or distribution as required by the license?

**Review by ljacomet — APPROVED** [review]
LGTM!

One question about license requirements though:

File: build-logic/basics/src/main/kotlin/gradlebuild.repositories.gradle.kts [hunk 2]
```diff
@@ -19,4 +19,11 @@ repositories {
         name = "Gradle libs"
         url = uri("https://repo.gradle.org/gradle/libs")
     }
+    maven {
+        name = "CHAMP libs"
+        url = uri("https://releases.usethesource.io/maven/")
```
Comment (ljacomet) [inline build-logic/basics/src/main/kotlin/gradlebuild.repositories.gradle.kts:24]: Given that we try to only use our own repo during build, this most likely needs to be added to our CI mirrors.
And a mirroring setup to be done

**Review by ljacomet — CHANGES_REQUESTED** [review]

File: build-logic/basics/src/main/kotlin/gradlebuild.repositories.gradle.kts [hunk 3]
```diff
@@ -19,4 +19,11 @@ repositories {
...
+        mavenContent {
+            includeModule("io.usethesource", "capsule")
```
Comment (blindpirate) [inline build-logic/basics/src/main/kotlin/gradlebuild.repositories.gradle.kts:26]: The parent pom (and modules) are also needed I guess?

Comment (bamboo) [reply build-logic/basics/src/main/kotlin/gradlebuild.repositories.gradle.kts]: Thanks, @blindpirate, that was it.

File: build-logic/basics/src/main/kotlin/gradlebuild.repositories.gradle.kts (hunk 2 above)
Comment (bamboo) [reply build-logic/basics/src/main/kotlin/gradlebuild.repositories.gradle.kts:24]: @blindpirate suggested adding the repo.

File: build-logic/dependency-modules/src/main/kotlin/gradlebuild/modules/extension/ExternalModulesExtension.kt (hunk 1 above)
Comment (bamboo) [reply build-logic/dependency-modules/src/main/kotlin/gradlebuild/modules/extension/ExternalModulesExtension.kt:189]: I don't know. @blindpirate, do you know?

**Comment by bamboo** [comment]
@bot-gradle test QFL

**Comment by bamboo** [comment]
@bot-gradle test ACC

**Comment by bamboo** [comment]
@bot-gradle test and merge

**Comment by blindpirate** [comment]
@bot-gradle test and merge

**Comment by bamboo** [comment]
@bot-gradle test and merge

**Pull request merged**